holidata/holidays/et-EE.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+from dateutil.easter import EASTER_WESTERN
+
+from .holidays import Locale
+
+
+class et_EE(Locale):
+    """
+    01-01: [NF] Uusaasta
+    02-24: [NF] Iseseisvuspäev, Eesti Vabariigi aastapäev
+    05-01: [NF] Kevadpüha
+    06-23: [NF] Võidupüha
+    06-24: [NF] Jaanipäev
+    08-20: [NF] Taasiseseisvumispäev
+    12-24: [NF] Jõululaupäev
+    12-25: [NF] Esimene jõulupüha
+    12-26: [NF] Teine jõulupüha
+    1 day before Easter: [NRV] Suur reede
```

Comment:
This is the friday before Easter, not 1 day. I think it's called Good Friday in english.

A nice overview can be found on this page: https://publicholidays.ee/easter/

Reply:
Woops, a miscalculation from my side... 😅